src/main-process/fetch/index.ts

```diff
@@ -8,7 +9,7 @@ const httpsAgent = new https.Agent({
 
 const _fetch = async (path: string, reqInit: RequestInit = {}): Promise<Response> => {
   const response = await fetch(path, {
-    agent: httpsAgent,
+    agent: app.isPackaged && httpsAgent,
```

Comment:
can you explain this one?

Reply:
When we are in dev env and pointing at the local server, the `httpsAgent` fails requests to `http`.
So `isPackaged` is true only when not in dev mode.